roles/openshift_sanitize_inventory/vars/main.yml

```diff
@@ -76,3 +76,5 @@ __warn_deprecated_vars:
   - 'openshift_hosted_metrics_storage_labels'
   - 'openshift_hosted_metrics_deployer_prefix'
   - 'openshift_hosted_metrics_deployer_version'
+
+__valid_openshift_releases: ['3.6', '3.7']
```

Comment:
1.5?

Reply:
3.8 as well. Based on https://github.com/openshift/openshift-ansible/pull/5904 the list does not have to suffice. Maybe replace the list with almost the same regexp as @mtnbikenc updated?

Reply:
Btw. why is the 3.7.0.0 valid value for the `openshift_release` variable?

Reply:
Several things to consider here.  Do we need to have a specific list here in the code?  Does this provide value to the user which is greater than the need to maintain this list for future releases?  If we do have this list, what should it contain, N-1, N (latest), N+1?  When I updated the regex in #5904 I was only correcting the expression to work as @sosiouxme intended.  Looking at our example inventory it appears we really just want a two-part 'generic' release defined for openshift_release, so the regex should probably on allow n.n.  There are other inventory variables that allow installing a particular version.